### PR TITLE
riff-distro image list

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/projectriff/riff/issues/756

### CODE REVIEW

**Comment by glyn** [comment]
This supersedes https://github.com/projectriff/riff/pull/843 as the PRs on which 843 was stacked have now been merged.

File: distro/commands/image.go [hunk 1]
```diff
@@ -57,3 +57,41 @@ func ImagePull(c *core.ImageClient) *cobra.Command {
...
+			}
+
+			commands.PrintSuccessfulCompletion(cmd)
+			return nil
+		},
+	}
+	command.Flags().StringVarP(&options.Manifest, "manifest", "m", "stable", "manifest to be searched; can be a named manifest (stable or latest) or a path of a manifest file")
+	command.MarkFlagFilename("manifest", "yml", "yaml")
+
+	command.Flags().StringVarP(&options.Images, "images", "i", "image-manifest.yaml", "path of the image manifest to be created")
```
Comment (ericbottard) [inline distro/commands/image.go]: The default value should be understood relative to the manifest file IMO

Comment (glyn) [reply distro/commands/image.go]: Agreed. Would you like to make that change as part of merging?

Comment (ericbottard) [reply distro/commands/image.go]: I can do that. And the inversion of `check` as well

**Comment by ericbottard** [comment]
Merged as cadf6ecf with a change to make the default of image-manifest be relative to the manifest

**Pull request closed without merging**